RecCalorimeter/src/components/ConstNoiseTool.h

```diff
@@ -34,34 +37,33 @@ class ConstNoiseTool : public GaudiTool, virtual public INoiseConstTool {
   virtual StatusCode finalize() final;
 
   /// Find the appropriate noise constant from the histogram
-  double getNoiseConstantPerCell(uint64_t aCellID);
+  double getNoiseRMSPerCell(uint64_t aCellID);
   double getNoiseOffsetPerCell(uint64_t aCellID);
 
 private:
-  std::map<uint,double> m_systemNoiseConstMap;
+  std::map<uint,double> m_systemNoiseRMSMap;
   std::map<uint,double> m_systemNoiseOffsetMap;
 
-  /// Cell thresholds (1Sigma) in GeV:
+  /// List of subdetector names (I think it is better to pass the names and retrieve the IDs in the C++)
```

Comment:
On drawback of this approach is that it will not work for a detector that passes the system id directly as a number in the xml. But probably no detector like this will use the tool. 

If we keep this approach, can you add as comment here that the name of the detector *must* match what is defined in DectDimentsion?